vignettes/data-transform-as-shiny-module.Rmd

```diff
@@ -116,7 +124,11 @@ my_transformators <- list(
     ui = function(id) {
       ns <- NS(id)
       tags$div(
-        numericInput(ns("n_rows"), "Number of rows to subset", value = 6, min = 1, max = 150, step = 1)
+        numericInput(
+          ns("n_rows"),
+          "Number of rows to subset",
+          value = 6, min = 1, max = 150, step = 1
+        )
       )
     },
     server = function(id, data) {
```

Comment:
The ending here seems a bit abrupt. Could we add something here to explain the example code a bit more, i.e., what's the UI like, now users can do xxx, etc.

Reply:
Added more content with this commit: 3888884e3d930ba099be637947ab2357b23bd1be